lib/readline.js

```diff
@@ -170,6 +174,12 @@ function Interface(input, output, completer, terminal) {
     throw new ERR_INVALID_ARG_VALUE('completer', completer);
   }
 
+  if (history === undefined) {
+    history = [];
+  } else if (!(history instanceof Array)) {
```

Comment:
```suggestion
  } else {
```

Reply:
Done!
Sorry, was to quick to change and did not think it through...